### DISCUSSION TITLE
How 2 make lvl difficulty show on gdps

### DISCUSSION THREAD

**Discussion by JumpCatGD** [zmxhawrhbg/gdrpc#19] [Q&A] [answered]
GDPS: 
![image](https://user-images.githubusercontent.com/89575019/193447024-01b8f105-3276-4e25-b90b-5e50bea91cb9.png)
Default GD RPC: ![image](https://user-images.githubusercontent.com/89575019/193446750-f4f27949-fa3f-4afd-8978-fd549193610c.png)

**Comment by qimiko** [comment, marked as answer]
I think the missing difficulty has to do with your translations, not the GDPS itself.

**Reply by JumpCatGD** [reply]
> I think the missing difficulty has to do with your translations, not the GDPS itself.

Maybe I need to add png difficulties to the discord app?

**Reply by qimiko** [reply]
If you haven't already done so, yes.
